src/workerd/api/node/async-hooks.h

```diff
@@ -29,12 +29,26 @@ namespace workerd::api::node {
 //   console.log(als.getStore());  // undefined
 class AsyncLocalStorage final: public jsg::Object {
  public:
-  AsyncLocalStorage(): key(kj::refcounted<jsg::AsyncContextFrame::StorageKey>()) {}
+  struct AsyncLocalStorageOptions {
+    jsg::Optional<jsg::JsRef<jsg::JsValue>> defaultValue;
+    jsg::Optional<kj::String> name;
```

Comment:
Any reason to why we are not storing this as JsString since it has a getter and it would avoid unnecessary copy whenever the getter is called?

Reply:
We have so many of these that I'll just update them at once or in a series of large batches when the time comes.